Source/WebCore/crypto/cocoa/CryptoAlgorithmAESGCMMac.cpp

```diff
@@ -54,14 +54,11 @@ ALLOW_DEPRECATED_DECLARATIONS_END
 #if HAVE(SWIFT_CPP_INTEROP)
 static ExceptionOr<Vector<uint8_t>> encryptCryptoKitAESGCM(const Vector<uint8_t>& iv, const Vector<uint8_t>& key, const Vector<uint8_t>& plainText, const Vector<uint8_t>& additionalData, size_t desiredTagLengthInBytes)
 {
-    // Protect against underflow in the subtraction below.
-    if (desiredTagLengthInBytes > CryptoAlgorithmAESGCM::DefaultTagLength / 8)
+    auto rv = PAL::AesGcm::encrypt(key.span(), iv.span(), additionalData.span(), plainText.span(), desiredTagLengthInBytes);
```

Comment:
Is the type of `rv` on the heap, or is that on the stack? Do we need to care about ref counting?

Reply:
swift moves back the value and then it will be cleaned up by C++.